.vscode/tasks.json

```diff
@@ -0,0 +1,25 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Compile Next.js Core (watch)",
```

Comment:
Do we want to mention this in the contributing docs? Also we should probably note that the other packages need to be built first before using this command that the types won't be generated with this command currently.